pom.xml

```diff
@@ -59,16 +72,57 @@
       <type>test-jar</type>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.druid</groupId>
+      <artifactId>druid-sql</artifactId>
+      <version>${druid.version}</version>
+      <type>test-jar</type>
+      <scope>test</scope>
+    </dependency>
     <dependency>
       <groupId>org.apache.druid</groupId>
       <artifactId>druid-benchmarks</artifactId>
       <version>${druid.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+      <scope>test</scope>
+    </dependency>
```

Comment:
I had to remove this `guava` dependency to get the project to build with `mvn package`. It gets inherited from Druid itself at compile, test, and runtime when removed. But when it's here, it's only present during test. I suppose including it as `provided` could also work.

Reply:
I've also seen some wierd exception - but this is not necessary ; I've removed a few more unnecessary deps